### PR TITLE
FindSimilarIncidents bug fix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24698

## Description
Casting to str in all `replace` occurences

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/NonSupported/Scripts/FindSimilarIncidents/FindSimilarIncidents.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, inci
 
     if ignore_closed:
         query += " and -status:Closed"
+        query += " and -status:Closed"
```
Comment (ShahafBenYakir) [inline Packs/NonSupported/Scripts/FindSimilarIncidents/FindSimilarIncidents.py:134]: What is this?

Comment (guyfreund) [reply Packs/NonSupported/Scripts/FindSimilarIncidents/FindSimilarIncidents.py:134]: mistake

**Comment by chris4490** [comment]
@guyfreund I see this was merged 2 days ago, but it wasn't in the 20.5.2 content release. Just wanted to make sure, this will be in the next content so I can communicate with customer.

**Comment by guyfreund** [comment]
@chris4490 yes, this will go into the next content release.

**Pull request merged**